### PR TITLE
String updates

### PR DESCRIPTION
**Pull request by esbrandt**

### CODE REVIEW

File: src/skin/tooltips.cpp [hunk 1]
```diff
@@ -810,11 +810,11 @@ void Tooltips::addStandardTooltips() {
     add("EffectSlot_parameter_link_type")
             << tr("Meta Knob Link")
             << tr("Set how this parameter is linked to the effect's Meta Knob.")
-            << tr("Empty bar: not linked")
-            << tr("Solid bar: moves with Meta Knob")
-            << tr("Left side: moves with left half of Meta Knob")
-            << tr("Right side: moves with right half of Meta Knob")
-            << tr("Left and right: moves across range with half of Meta Knob and back with the other half");
+            << tr("Inactive: parameter not linked")
+            << tr("Active: parameter moves with Meta Knob")
```
Comment (Be-ing) [inline src/skin/tooltips.cpp]: :+1:

**Comment by uklotzde** [comment]
Thanks for this polishing work! Another hint that we should focus on releasing something, finally 😉

**Comment by daschuer** [comment]
LGTM! @Be merge? 

Testing this I found that we have a random tab order in our preferences, but that is an other issue.

**Comment by Be-ing** [comment]
Yes LGTM

**Comment by daschuer** [comment]
Btw: Thank you @esbrandt!

**Pull request merged**